test/test845.js

```diff
@@ -21,4 +21,22 @@ describe('Test ' + test + ' - use NOW() function', function () {
 		//02/25/22
 		assert(/\d{2}\/\d{2}\/\d{2}/.test(res[0].conv));
 	});
+
+	it('3. NOW() with point', function () {
+		var currentSeparator = alasql.options.nowdateseparator;
+		alasql.options.nowdateseparator = '.';
+		var res = alasql('SELECT NOW() AS now');
+		//2022.02.25 19:21:27.839
+		assert(/\d{4}.\d{2}.\d{2} \d{2}:\d{2}:\d{2}.\d{3}/.test(res[0].now));
+		alasql.options.nowdateseparator = currentSeparator;
+	});
+
+	it('4. CONVERT with NOW() with point as an argument', function () {
+		var currentSeparator = alasql.options.nowdateseparator;
+		alasql.options.nowdateseparator = '.';
+		var res = alasql('SELECT CONVERT(STRING,NOW(),1) AS conv');
+		//02/25/22
+		assert(/\d{2}\/\d{2}\/\d{2}/.test(res[0].conv));
+		alasql.options.nowdateseparator = currentSeparator;
+	});
 });
```

Comment:
Unfortunately, even a large number of tests does not guarantee stable operation, because the result of new Date() depends on the browser. For example, the following code:
`new Date("2022.01.01 10:22:22") // returns 'Invalid Date' in Firefox
new Date("2022-10-13 21:43:24.729")  //returns 'Invalid Date' in Safari 13.1`

But both these cases will return correct Date in Nodejs

Reply:
The main purpose of this commit is to correctly convert the string returned by the Now function to a date with the separator '-' or '.'

Reply:
I changed the default parameter `alasql.options.nowdateseparator = '-';`
then ran the tests, all of them were successful, except test 845 '1. NOW()', but this is expected.

If we could break something, then the tests would not be completed successfully.

Reply:
Its a breaking change. We will bump the major version